src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -52,6 +53,36 @@
 
     private final static Logger logger = new LoggerImpl(ColibriConferenceImpl.class.getName());
 
+    private static Pair<Map<String, List<SourcePacketExtension>>, Map<String, List<SourceGroupPacketExtension>>>
```

Comment:
Can this type get named?  It's pretty verbose.  (Assuming it's going to stick around for the final version of the PR.)

Reply:
Yes, for sure.